force-app/main/default/lwc/messagePopup/messagePopup.js

```diff
@@ -94,10 +94,13 @@ export default class MessagePopup extends LightningElement {
         }
     }
 
+    timeoutInstance
```

Comment:
What is this line doing here? 🤔

Reply:
This is a LWC property/variable without @track attribute; I have declared in between instead of in start. i updated the code.